docs/site/development/adding-locales.md

```diff
@@ -6,37 +6,113 @@ title: Adding a new locale to CLDR
+- Consider whether the locale needs parent locale information (which it probably won't)
+	- If parent inheritance can be determined by truncating subtags (and then falling back to root), no parent locale information is needed.
+In the majority of cases this will be true.
+	- If the locale is written in the non-preferred script for the language, it needs to have the parent specified as root,
+for example, zh_Hant and sr_Latn need to fall back to root (not to zh and sr which are in different scripts).
+	- Similarly, en_GB inherits from en_001 not from en, and nb inherits from no, not root.
+	- Make the appropriate entry to a parentLocale element in **common/supplemental/supplementalData.xml**
+- Verify or add time cycle information in **common/supplemental/supplementalData.xml**
+	- In the majority of cases no changes will be needed.
+	- In general terms, check that, under the `<timeData>` element,
+the `<hours>` element that has an attribute `"regions"` that contains the region for this locale,
+also has a `"preferred"` attribute that corresponds to the time cycle selected for the locale.
+	- NB: Even though the attribute is `"regions"`, it can contain a region or a locale.
+Consider the example of `"fr_CA"` in the `"regions"` attribute of an `"hours"` element where the `"preferred"` attribute is `"H"`,
+in contrast to `"CA"` which is in the `"regions"` attribute of an `"hours"` element where `"preferred"` attribute is `"h"`.
+This corresponds to Canadian French (fr_CA) using 24-hour time even though 12-hour time is preferred for Canada (region CA).
+	- For the locale submission for `"pap"` (ticket [CLDR-14872](https://unicode-org.atlassian.net/browse/CLDR-14872)
+and pull request [PR#2542](https://github.com/unicode-org/cldr/pull/2542)),
+the region `"CW"` did not appear in the `"timeData"` information and needed to be added to the appropriate location.
+
+#### Optional additions
+- Plural rules are not needed until the locale needs to move beyond Basic level,
+but it is **strongly recommended** that plural rule information be collected as early as possible.
+   - The addition of new plural forms to a language after data collection at higher levels has begun
+will cause the coverage percentage to drop significantly until the data for plurals can be provided.
+The addition of new plural forms can also be disruptive for any localized applications which are relying on CLDR plural data.
+   - If plural rule information is available, it can be added at the same time the locale is established.
+Add the rules to **common/supplemental/plurals.xml** and **common/supplemental/ordinals.xml**.
+- Similarly day period rules can be added to **common/supplemental/dayPeriods.xml**.
+
+## Submit the pull request
+
+- If requested, add to vendor targets in **tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt**,
+and to Cldr where resources are committed.
```

Comment:
```suggestion
and to CLDR where resources are committed.
```

Reply:
"Cldr" was in the original text. I think it refers to the entry in Locales.txt (where it is "Cldr" rather than "CLDR"). I'll go ahead and include this change. It seems pretty clear either way.